packages/docs/docs/contributing/run-the-stack.md

```diff
@@ -0,0 +1,149 @@
+
+:::
+
+## Install
+
+Our monorepo uses [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces), so installing dependencies for all projects is done using normal `npm install`:
+
+```sh
+cd medplum
+npm ci
+```
+
+## Build and Test
+
+We provide convenience scripts to perform a full build and test:
+
+```sh
+./scripts/build.sh
+```
+
+This will do the following:
+
+- Install npm dependencies if not already installed
+- Build all packages
+- Run all tests
+- Run linter
+
+## Run
+
+### Background services
+
+The Medplum Stack requires the following services to be running in your environment:
+
+- [PostgreSQL](https://www.postgresql.org/) for the primary database
+- [Redis](https://redis.com/) for caching and job queueing
+
+When running this services on your local machine you can either use Docker (recommended) or install them directly onto your machine.
+
+#### Using Docker (Recommended)
+
+Use the supplied `docker-compose.yml` file to run PostgreSQL and Redis background services. These services will be deployed with all necessary medplum configurations and database migrations.
+
+From your root `medplum` directory run
+
+```sh
+docker-compose up
+```
+
+This will:
+
+1. Start the PostgreSQL server in a container
+2. Set up the appropriate configurations (see [postgres.conf](https://github.com/medplum/medplum/postgres/postgres.conf))
+3. Create two databases for testing: `medplum` and `medplum_test` (see [init_test.sql](https://github.com/medplum/medplum/postgres/init_test.sql))
+
+When `docker-compose` completes, you should see something like this in your terminal:
+
+```bash
+medplum-postgres-1  | PostgreSQL init process complete; ready for start up.
+medplum-postgres-1  |
+medplum-postgres-1  | 2022-07-29 00:37:44.639 GMT [1] LOG:  starting PostgreSQL 12.10 (Debian 12.10-1.pgdg110+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
+medplum-postgres-1  | 2022-07-29 00:37:44.639 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
+medplum-postgres-1  | 2022-07-29 00:37:44.639 GMT [1] LOG:  listening on IPv6 address "::", port 5432
+medplum-postgres-1  | 2022-07-29 00:37:44.642 GMT [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
+medplum-postgres-1  | 2022-07-29 00:37:44.650 GMT [88] LOG:  database system was shut down at 2022-07-29 00:37:44 GMT
+medplum-postgres-1  | 2022-07-29 00:37:44.654 GMT [1] LOG:  database system is ready to accept connections
+```
+
+#### Deploying manually
+
+If you'd prefer to install the dependencies directly, you can find installation instructions for the required services below:
+
+1. [Install PostgreSQL](https://www.postgresql.org/download/)
+2. [Install Redis](https://redis.io/download)
+
+After that, you will have to update the file `packages/server/medplum.config.json`
+
+```js
+  "database": {
+    "host":  // YOUR POSTGRESQL HOST
```

Comment:
Not a big deal, but this will cause syntax errors when rendered.

If you add empty strings, it might render better.  I.e.:

```js
  "database": {
    "host": "", // YOUR POSTGRESQL HOST
```